eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Translation.hs

```diff
@@ -47,6 +47,8 @@ import Data.Maybe (fromMaybe)
 -- being total. Do not change it!
 --------------------------------------------------------------------------------
 
+type MaryEra = ShelleyMAEra 'Mary
+
```

Comment:
MaryEra  is used 20 times in this file.  But because of the refatoring, it cannot be imported from Cardano.Ledger.Mary (where it was previously defined) without causing cyclic imports. So I now define it in Cardano.Ledger.ShelleyMA.Era, And it is imported from there (both here and inside of Cardano.Ledger.Shelley)

Reply:
I used the same trick for both MaryEra and AllegraEra.